website/src/queries-logic.js

```diff
@@ -207,27 +204,22 @@ function add_fork_elements(forkdata_array, user, repo, parentDefaultBranch) {
       head: `${extract_username_from_fork(currFork.full_name)}:${currFork.default_branch}`
     });
     const onSuccess = (responseHeaders, responseData) => {
-      if (responseData.total_commits <= AHEAD_COMMITS_FILTER) {
-        NEW_ROW.remove();
-        if (tableIsEmpty(table_body)) {
-          setMsg(UF_MSG_EMPTY_FILTER);
+      if (responseData.total_commits > AHEAD_COMMITS_FILTER) {
+        datum['ahead_by'] = responseData.ahead_by;
+        datum['ahead_url'] = responseData.html_url;
+        datum['behind_by'] = responseData.behind_by;
+        datum['behind_url'] = getBehindUrl(responseData.html_url);
+        datum['pushed_at'] = getOnlyDate(currFork.pushed_at);
+        TABLE_DATA.push(datum);
+
+        if (typeof is_useful_fork === 'function') {
```

Comment:
Excuse my ignorance: why do we need to check this is a function? (i.e. when would this not be a function?)

Reply:
When the function is called without parameter of `is_useful_fork` (so it's undefined). And I think to check whether it's a function is a little better to check whether it's undefined because we only need function.

You will see `undefined is not a function` error in console if we remove the check.